tls/platform.h

```diff
@@ -125,8 +125,8 @@ int mbedtls_platform_set_calloc_free( void * (*calloc_func)( size_t, size_t ),
                               void (*free_func)( void * ) );
 #endif /* MBEDTLS_PLATFORM_FREE_MACRO && MBEDTLS_PLATFORM_CALLOC_MACRO */
 #else /* !MBEDTLS_PLATFORM_MEMORY */
-#define mbedtls_free       free
-#define mbedtls_calloc     calloc
+#define mbedtls_free       ttls_free
+#define mbedtls_calloc     ttls_calloc
```

Comment:
Why do we need the `ttls_*` calls? It seems we can just define them as `kfree()` and `kzalloc()` correspondingly.